### ISSUE TITLE
Group created with Concierge automatically gets pinned

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#13451]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Actions Performed:

1. Click on :global-create: > New group
2. Select Concierge and any other users
3. Click Create group
4. Switch to any other chat
5. Switch back to the group and observe that the group is pinned

## Expected Result:
Group is not pinned automatically

## Actual Result:
Group is pinned automatically

## Workaround:
unknown

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.2.37-2
**Reproducible in staging?:** y
**Reproducible in production?:** y
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/43996225/206590162-9fc59dbd-7496-4133-8aa8-3edc626ddbe2.mov

https://user-images.githubusercontent.com/43996225/206590171-9175d308-93f3-4b16-9b5b-604dba6fd3b0.mp4

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @adeel0202 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1670486631874069

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by trjExpensify** [comment]
Confirmed I can reproduce this:

<img width="1511" alt="image" src="https://user-images.githubusercontent.com/16232057/206739495-69e64faa-18f0-45c6-89b7-ffa05dd32dc9.png">

I took a cursory look to see if I can find anything that modified this recently, bit didn't find much. Here's the PR that introduced pinning the _Concierge 1:1 DM_ for new signups: https://github.com/Expensify/Web-Expensify/pull/32271

Marking this one as `internal` as it is set on the back-end.

**Comment by trjExpensify** [comment]
No update since Friday, Melvin.

**Comment by trjExpensify** [comment]
@tylerkaraszewski are you going to be able to progress this one given the focus on #deprecate-sequence-numbers, or shall we ask in #engineering-chat if someone else to take it over? There are probably people looking for a WAQ issue, such that we can try to keep this moving while you focus on that.

**Comment by trjExpensify** [comment]
^^ Question still stands.

**Comment by neil-marcellini** [comment]
@tylerkaraszewski I'm happy to pick this up if you need to focus on sequence numbers.

**Comment by tylerkaraszewski** [comment]
Someone can take it.

**Comment by trjExpensify** [comment]
Awesome, sounds good! @neil-marcellini passing this over to yah 👍

**Comment by neil-marcellini** [comment]
Thanks. I'll get to this tomorrow as my top priority issue.

**Comment by neil-marcellini** [comment]
👀

**Comment by neil-marcellini** [comment]
I'm able to reproduce the issue on the latest main branches on dev.

**Comment by neil-marcellini** [comment]
Quite simply, whenever we create a chat report that includes concierge we send a welcome message from concierge and we pin it [here](https://github.com/Expensify/Web-Expensify/blob/2b23cee2288c0cc0379456157413bce0232a9f9d/lib/ReportAPI.php#L689).

Do we want concierge to send the welcome message in group chats? I think we probably want to keep that behavior and then only pin the chat if it's an individual concierge chat. I'll put up a PR with that change.

**Comment by neil-marcellini** [comment]
Looking at the Slack thread it sounds like we no longer want to automatically pin concierge chats. I asked for some more consensus in Slack [here](https://expensify.slack.com/archives/C03U7DCU4/p1671753109798699). While I wait for responses I'll start working on the PR and I can always tweak the behavior based on what we decide.

**Comment by neil-marcellini** [comment]
I have a fix under review. I'll be OOO tomorrow and Monday. The fix won't unpin group chats with concierge that have already been created, but I think that's fine because the user can manually unpin them. If it's really a problem we could run a CQ to fix it.

**Comment by trjExpensify** [comment]
Thanks, @neil-marcellini. Looks like the Web-E PR is on prod, right? 

>The fix won't unpin group chats with concierge that have already been created, but I think that's fine because the user can manually unpin them. If it's really a problem we could run a CQ to fix it.

I agree with this, we don't need to apply the fix retroactively.

**Comment by neil-marcellini** [comment]
Yeah, the Web-E PR is on prod and this is fixed. Here's the checklist and I'll close this out.

**BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:

- [ ] [@neil-marcellini] The PR that introduced the bug has been identified. Link to the PR:
N/A missed during the initial implementation / it's always been like this
- [ ] [@neil-marcellini] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
N/A
- [ ] [@neil-marcellini ] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:
I don't think we need to start a conversation about this bug since it was something we didn't think of during the initial implementation.
- [ ] [@trjExpensify] A [regression test](https://expensify.testrail.io/index.php?/dashboard) has been added or updated so that the same bug will not reach production again. Link to the GH issue for creating the test here:
IMO I don't think we need a regression test for this because I added some integration tests which will automatically catch this bug in the future.
